staging/src/k8s.io/sample-controller/pkg/client/clientset/versioned/clientset.go

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2017 The Kubernetes Authors.
+Copyright 2017 The Kubernetes sample-controller Authors.
```

Comment:
This needs to be `Copyright 2017 The Kubernetes Authors.` everywhere and not `Copyright 2017 The Kubernetes <repo> Authors.`

Reply:
Hmm but these are generated files...

Reply:
Not sure why these have been marked as outdated by GitHub.... 🤔 

I copied this from the sample-apiserver repo: https://github.com/sttts/kubernetes/blob/sttts-codegen-scripts/staging/src/k8s.io/sample-apiserver/hack/custom-boilerplate.go.txt